### PR TITLE
make a few more classes/methods non-static

### PR DESCRIPTION
**Pull request by thiagowfx**
Bug: #392
Docs: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-extraneous-class.md

### CODE REVIEW

File: src/bidiMapper/BidiServer.ts [hunk 1]
```diff
@@ -85,10 +85,14 @@ export class BidiServer extends EventEmitter<BidiServerEvents> {
       flatten: true,
     });
 
+    await server.topLevelContextsLoaded();
+    return server;
+  }
```
Comment (thiagowfx) [inline src/bidiMapper/BidiServer.ts:90]: This is a snippet from https://github.com/GoogleChromeLabs/chromium-bidi/pull/400

File: src/bidiServer/bidiServerRunner.ts [hunk 2]
```diff
@@ -25,6 +25,20 @@ const debugInternal = debug('bidiServer:internal');
 const debugSend = debug('bidiServer:SEND ►');
 const debugRecv = debug('bidiServer:RECV ◀');
 
+async function getHttpRequestPayload(
+  request: http.IncomingMessage
+): Promise<string> {
+  return new Promise((resolve) => {
+    let data = '';
+    request.on('data', (chunk) => {
```
Comment (OrKoN) [inline src/bidiServer/bidiServerRunner.ts:33]: let's add a reject on error?

**Review by OrKoN — APPROVED** [review]

File: src/bidiServer/bidiServerRunner.ts (hunk 2 above)
Comment (thiagowfx) [reply src/bidiServer/bidiServerRunner.ts:33]: PTAL

**Review by OrKoN — APPROVED** [review]

File: src/bidiTab/mapperTabPage.ts [hunk 3]
```diff
@@ -16,12 +16,6 @@
  */
 import {LogType} from '../utils/log.js';
 
-declare global {
-  interface Window {
-    MapperTabPage: MapperTabPage;
```
[the remark this answers is not in the document]
Comment (thiagowfx) [reply src/bidiTab/mapperTabPage.ts:21]: https://github.com/GoogleChromeLabs/chromium-bidi/issues/433

**Pull request merged**